src/QuantumSdk/README.md

```diff
@@ -88,111 +88,113 @@ The content of the file should be similar to the following, with `Package_Name`
 
-- `QsharpCompile`:    
+- `QsharpCompile`:
 Contains all Q# source files included in the compilation.
 
 # Sdk Packages #
 
 A NuGet package of type `Sdk` enjoys certain privileges in terms of when and how its content is loaded.
-To understand how the content in this package works it is useful to understand how the properties, item groups, and targets defined in the Sdk are combined with those defined by a specific project. 
-The order of evaluation for properties and item groups is roughly the following: 
+To understand how the content in this package works it is useful to understand how the properties, item groups, and targets defined in the Sdk are combined with those defined by a specific project.
+The order of evaluation for properties and item groups is roughly the following:
 
 - Properties defined in \*.props files of the Sdk
 - Properties defined or included by the specific project file
 - Properties defined in *.targets files of the Sdk
 - Item groups defined in *.props files of the Sdk
 - Item groups defined or included by the specific project file
-- Item groups defined in *.targets files of the Sdk  
+- Item groups defined in *.targets files of the Sdk
 
-Similar considerations apply for the definition of targets. MSBuild will overwrite targets if multiple targets with the same name are defined. In that case, the target is replaced in its entirety independent on whether the values for `DependsOn`, `BeforeTarget`, and `AfterTarget` match - i.e. those will be overwritten. However, a target can be "anchored" by the surrounding targets' specifications of their dependencies, see e.g. the defined `BeforeCsharpCompile` target. 
+Similar considerations apply for the definition of targets. MSBuild will overwrite targets if multiple targets with the same name are defined. In that case, the target is replaced in its entirety independent on whether the values for `DependsOn`, `BeforeTarget`, and `AfterTarget` match - i.e. those will be overwritten. However, a target can be "anchored" by the surrounding targets' specifications of their dependencies, see e.g. the defined `BeforeCsharpCompile` target.
 
 ## Load context in .NET Core
 
-To avoid issues with conflicting packages, we load each Q# compiler extension into its own context. For more information, see the Core CLR [design docs](https://github.com/dotnet/coreclr/blob/master/Documentation/design-docs/assemblyloadcontext.md). 
+To avoid issues with conflicting packages, we load each Q# compiler extension into its own context. For more information, see the Core CLR [design docs](https://github.com/dotnet/coreclr/blob/master/Documentation/design-docs/assemblyloadcontext.md).
 
 ## Known Issues ##
 
 The following issues and PRs may be of interest when using the Sdk:
-> https://github.com/NuGet/Home/issues/8692    
```

Comment:
I am not sure if a formatter somehow did the whitespace removal, but could you please keep all whitespace in the markdown file as is, since it is needed...

Reply:
Ah, thanks for the catch! A formatter must have done so, but I'll revert that change.